quickstarts/microsoft.maintenance/maintenance-configuration-assignments/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "assignmentName": {
+            "value": "GEN-UNIQUE"
+        },
+        "vmName": {
+            "value": "GEN-UNIQUE"
+        },
+        "maintenanceConfigurationName": {
+            "value": "GEN-UNIQUE"
```

Comment:
This is just coming as one string. Eg: abc
But it should come something like
/subscriptions/edf2cef4-bc47-4278-b4f8-cfc65f25dfd8/resourcegroups/abcd/providers/Microsoft.Maintenance/maintenanceConfigurations/mc_name